apps/typeform/lambda/app.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const path = require('path');
+const express = require('express');
+const fetch = require('node-fetch');
+const cors = require('cors');
+const handleForms = require('./forms-handler');
+const handleWorkspaces = require('./workspaces-handler');
+const fetchAccessToken = require('./fetch-access-token');
+
+const deps = {
+  fetch
+};
+
+const app = express();
+
+const FRONTEND = path.dirname(require.resolve('typeform-frontend'));
+
+app.use(cors());
```

Comment:
CORS shouldn't be needed as we serve both frontend and the backend from the same domain.

Reply:
Good catch! Was using CORS for development